### ISSUE TITLE
support 3.73?

### ISSUE THREAD

**Issue by putra1990** [theheroGAC/TF-Card-Plugin-Tool#4]
support 3.73 hencore2?
thank

**Comment by theheroGAC** [comment]
yes now work on 3.73 https://github.com/theheroGAC/TF-Card-Plugin-Tool/releases/download/2.27/TF.Card.Plugin.Tool.vpk
